client/landing/stepper/declarative-flow/internals/types.ts

```diff
@@ -102,6 +102,12 @@ export type Flow = {
 	variantSlug?: string;
 	title?: string;
 	classnames?: string | [ string ];
+	/**
+	 * Optional flag to indicate if the flow is a signup flow.
```

Comment:
The flag is no longer optional.
```suggestion
	 * Flag to indicate if the flow is a signup flow.
```

Reply:
Updated in edb04500ed543fb96d408c68cfdef6e77631a7c6.